examples/layer-browser/src/examples/core-layers.js

```diff
@@ -175,7 +177,11 @@ const ScatterplotLayerExample = {
     pickable: true,
     radiusScale: 30,
     radiusMinPixels: 1,
-    radiusMaxPixels: 30
+    radiusMaxPixels: 30,
+    parameters: {
+      blend: true,
+      blendEquation: GL.MAX
```

Comment:
The weird blend effect was caused by improper blend functions of the alpha channel.
But this is also incorrect.
First, we should change the default blending mode instead of in the example.
Second, it should be
```
gl.blendFuncSeparate(gl.SRC_ALPHA, gl.ONE_MINUS_SRC_ALPHA, gl.ONE, gl.ONE_MINUS_SRC_ALPHA);
gl.blendEquationSeparate(gl.FUNC_ADD, gl.FUNC_ADD);
```

Reply:
I think Ib's intension is to remove the cascaded blending effects of circles, make all the circles melted together, doubt that can be resolved by a regular alpha blending.

Reply:
Because we use this app for debugging, it is important that it reflects the default behavior of deck.gl. If we believe the default behavior does not provide the best user experience, then let's change the defaults. We've seen hidden customization in examples causing more confusion for the users.

Reply:
Also `gl.MAX` is [WebGL2 only](https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/blendEquation) so this will not work in Safari.

Reply:
> Also gl.MAX is WebGL2 only so this will not work in Safari.

FWIW, this is available as a fairly commonly supported [extension](https://developer.mozilla.org/en-US/docs/Web/API/EXT_blend_minmax) under WebGL1. luma.gl transparently uses the extension when its available under WebGL1. It would be my guess that it is available in Safari.

Reply:
Just tested on desktop Safari, it works.

Reply:
> It should be
> ```
gl.blendFuncSeparate(gl.SRC_ALPHA, gl.ONE_MINUS_SRC_ALPHA, gl.ONE, gl.ONE_MINUS_SRC_ALPHA);
gl.blendEquationSeparate(gl.FUNC_ADD, gl.FUNC_ADD);
```

We added recently a new `DeckGL` prop `initWebGLParameters` that defaults to false (it is supposed to be set to default true in next major release). This allows us to play with changing the default parameter set already now without breaking apps that don't opt in.

I tried adding the recommended changes to `initWebGLParameters`. It changed the color quite a bit, but did not solve the overdraw and Z fighting issues.

<img width="691" alt="screen shot 2017-10-16 at 3 44 27 pm" src="https://user-images.githubusercontent.com/7025232/31638556-1f7c7f6a-b289-11e7-8a50-e03403cf0358.png">

Reply:
Updated formatting of previous post

Reply:
I don't see why overlapping is an issue, if you don't like it you can just set layer opacity to 1.
Z fighting is something we should definitely resolve, and hiding it from our primary debugging portal sounds like the wrong approach.